aiida/tools/dbimporters/baseclasses.py

```diff
@@ -191,7 +191,7 @@ class DbEntry(object):
     _license = None
 
     def __init__(self, db_name=None, db_uri=None, id=None,
-                 version=None, extras={}, uri=None):
+                 version=None, lic=None, extras={}, uri=None):
```

Comment:
I suggest using full variable name ``license`` for clarity. I am aware of ``license()`` function in Python, but ``id()`` is also a function, thus there should not be a problem.

Reply:
In fact I am changing this now...does not work.

Reply:
Fixed.